src/zenml/zen_server/deploy/helm/templates/server-deployment.yaml

```diff
@@ -61,12 +61,24 @@ spec:
               value: {{ .Values.zenml.defaultProject | quote }}
             - name: ZENML_DEFAULT_USER_NAME
               value: {{ .Values.zenml.defaultUsername | quote }}
-            {{- if .Values.zenml.database.url }}
```

Comment:
This should stay in to conditionally configure `ZENML_STORE_TYPE` to `sql`

Reply:
If I'm not mistaken, the entire initContainer is wrapped in a `{{- if .Values.zenml.database.url }}` which is why I removed this duplicate

Reply:
Oh, right :)